draft-ietf-httpbis-http2bis.xml

```diff
@@ -2159,7 +2160,7 @@
           the PUSH_PROMISE.
         </t>
         <t>
-          A PUSH_PROMISE frame modifies the connection state in two ways.  First, the inclusion of a <xref target="HeaderBlock">header block</xref> potentially modifies the state maintained for
+          A PUSH_PROMISE frame modifies the connection state in two ways.  First, the inclusion of a <xref target="HeaderBlock">field block</xref> potentially modifies the state maintained for
           header compression.  Second, PUSH_PROMISE also reserves a stream for later use, causing the
```

Comment:
Is 'header compression' still what you want here?

Reply:
I think that it is still OK to call it header compression, especially as we aren't revising RFC 7541.  It is a little janky and it probably needs better explanation, but I don't see a better alternative.

Reply:
I think we can call it field section compression everywhere in this document, and explain in the terms section how 7541 Header Compression applies?